pull_request_template.md

```diff
@@ -1,39 +1,40 @@
 ## Overview
+<!-- Please provide a brief summary of the changes proposed in this pull request. Make sure branch name matches the ticket below. -->
 
-Please provide a brief summary of the changes proposed in this pull request. Make sure branch name matches the ticket below.
 
 ## Ticket
+<!-- [NEX-XXXX](URL) -->
 
-[NEX-XXXX](URL)
 
 ## Cloud Changes
+<!-- List any changes made to cloud infrastructure or configurations, including the reasoning behind the changes. -->
 
-List any changes made to cloud infrastructure or configurations, including the reasoning behind the changes.
 
 ## Wireless Team Changes
+<!-- List any changes made to embedded systems or device configurations, including the reasoning behind the changes. -->
 
-List any changes made to embedded systems or device configurations, including the reasoning behind the changes.
 
 ## User Interface
+<!-- List any changes made to the application code, including the reasoning behind the changes. -->
 
-List any changes made to the application code, including the reasoning behind the changes.
 
 ## Test Plan
+<!-- Place Test Specifications (if they were done) -->
 
-Place Test Specifications (if they were done)
 
 ## Additional Notes
+<!-- Provide any additional notes or information that may be relevant to this pull request. -->
+<!-- Please include any relevant links or resources, such as related issues or documentation. -->
 
-Provide any additional notes or information that may be relevant to this pull request.
-Please include any relevant links or resources, such as related issues or documentation.
 
 ## Procedures
-- I have updated the Title with Jira Ticket Id
-- I have listed critical dependencies in Additional Notes / Overview
-- I have read the [contributing guidelines](CONTRIBUTING.md) and my changes adhere to them
-- I have scheduled a formal code review with atleast 2 others, and applied feedback
-- I have worked with QE to generate relevant test cases
-- I have added any relevant documentation or comments to the code
-- I have reviewed the changes and signed off on them
-- I have added reviewers to the pull request
-- I have added ticket information to this Pull Request
+
+- [ ] I have updated the Title with Jira Ticket Id
```

Comment:
We had feedback from the team that checkboxes were a bit too much
Maybe we could try it out and switch back 

Perhaps add a comment to say how to check the items? Something like this:
<!-- Please place an x into the boxes, without spaces, to 'check' the box. [x] -->

Reply:
I thought it would be interactive, but it's not, so I don't think it doesn't make sense to make it into a check list anymore although personally it is more aesthetically pleasing. Especially if it's just a mental checklist for people to go through and not a requirement to have checked off all the boxes.

Reply:
Yeah it's not interactive - and doing the action of placing an 'x' made it a bit more explicit but requires a bit more effort
Let's remove the boxes, for now if you are okay with it

Reply:
Done